entity-types/infra-redisinstance/definition.yml

```diff
@@ -4,23 +4,29 @@ type: REDISINSTANCE
 synthesis:
   # Apart from this rule REDISINSTANCE entities are also created from infra pipeline.
   rules:
-    # Redis receiver from otel collector.
-    # Currently the receiver doesn't have an identifier, added redis.instance.id with a processor for POC
-  - identifier: redis.instance.id
-    name: redis.instance.id
+  - compositeIdentifier:
+      separator: ":"
+      attributes:
+        - server.address
+        - server.port
+    name: server.address
     encodeIdentifierInGUID: true
     conditions:
       - attribute: eventType
         value: MetricRaw
+      # All metrics from the receiver starts with the 'redis.' prefix.
       - attribute: metricName
         prefix: redis.
       - attribute: instrumentation.provider
         value: opentelemetry
+      # This filters only metrics coming from redis receiver, given that metrics
+      # could deffer between different runtime receivers. 
       - attribute: otel.library.name
         value: otelcol/redisreceiver
     tags:
       # Default resource attributes
-      redis.instance.id:
+      # redis.instance.id:  # TODO: This is still not reported by the receiver.
+      newrelic.redis.instance_name:
```

Comment:
If we are not reporting it yet I would add it for now

Reply:
You are right. It was a leftover I did not realize during the change of the ID. I fixed it on the rebase.